libraries/standard/mqtt/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Include filepaths for source and include.
+include(filePaths.cmake)
+
+# MQTT library target.
+add_library( mqtt
+             ${MQTT_SOURCES}
+             include/mqtt.h
+             include/mqtt_lightweight.h )
+
+# MQTT public include path.
+target_include_directories( mqtt PUBLIC ${MQTT_INCLUDE_PUBLIC_DIRS} )
```

Comment:
How about the private include directory? 
`target_include_directories( mqtt PRIVATE {CMAKE_CURRENT_LIST_DIR}/src/private )`

Reply:
Oops... Forgot to include this line. Thanks!
```cmake
# MQTT private include path.
target_include_directories( mqtt PRIVATE ${MQTT_INCLUDE_PRIVATE_DIRS} )
```